src/main/java/cz/smarteon/loxone/app/state/events/LockedEvent.java

```diff
@@ -0,0 +1,17 @@
+package cz.smarteon.loxone.app.state.events;
+
+import cz.smarteon.loxone.app.state.Locked;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class LockedEvent {
+
+    @Builder.Default
+    Locked locked = Locked.NO;
+
+    String reason;
```

Comment:
should be private?

Reply:
The @Value annotation marks the fields private at compile time, i could add it to the source code, but it is not needed because lombok adds it